ci/stress: ensure that the distributed sends will be stopped

azat opened the pull request:
It is possible that this query will fail, due to `CANNOT_SCHEDULE_TASK` (due to `cannot_allocate_thread_fault_injection_probability`) and increasing it - will not help (even 30 retries failed with async load), so let's simply disable this fault injections for the last server (the one that ensures that the tables are correct after stressing)

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Follow-up for: https://github.com/ClickHouse/ClickHouse/pull/72558

azat committed:
ci/stress: retry "SYSTEM STOP DISTRIBUTED SENDS" in case of failures

It is possible that this query will fail, due to CANNOT_SCHEDULE_TASK
(due to cannot_allocate_thread_fault_injection_probability), and since
we cannot change this setting on a query basis, let's simply retry the
query, since otherwise the shutdown can take too much time.

robot-ch-test-poll commented:
<!-- automatic status comment for PR #75670 from azat/ClickHouse:ci/ensure-distributed-sends-disabled -->
*This is an automated comment for commit 0a472ed2df545463c1443425cd55f2d9ab4a6d6f with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/75670/0a472ed2df545463c1443425cd55f2d9ab4a6d6f/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Builds</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/ba50bd8970cd21df5e6d5963a8e2741661fe6012/builds/report.html">✅ success</a></td></tr>
<tr><td>ClickBench</td><td>Runs <a href="https://github.com/ClickHouse/ClickBench/">ClickBench</a> with instant-attach table</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/clickbench__aarch64_.html">✅ success</a></td></tr>
<tr><td>Compatibility check</td><td>Checks that <b>clickhouse</b> binary runs on distributions with old libc versions. If it fails, ask a maintainer for help</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/compatibility_check__aarch64_.html">✅ success</a></td></tr>
<tr><td>Docker keeper image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/docker_keeper_image.html">✅ success</a></td></tr>
<tr><td>Docker server image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/docker_server_image.html">✅ success</a></td></tr>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/0bfda2c21d37fdc82a9bff9ccadade7fe66e4d63/docs_check.html">✅ success</a></td></tr>
<tr><td>Install packages</td><td>Checks that the built packages are installable in a clear environment</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/install_packages__aarch64_.html">✅ success</a></td></tr>
<tr><td>Integration tests</td><td>The integration tests report. In parenthesis the package type is given, and in square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/integration_tests__aarch64__[1_6].html">✅ success</a></td></tr>
<tr><td>Performance Comparison</td><td>Measure changes in query performance. The performance test report is described in detail <a href="https://github.com/ClickHouse/ClickHouse/tree/master/docker/test/performance-comparison#how-to-read-the-report">here</a>. In square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/performance_comparison__release__[1_4]/report.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/95f5372a31bed4635899282330e1f1cf3749fa50/unit_tests__asan_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

Algunenano reviewed (approved):
:pray:

azat commented:
>[Stress test (debug)](https://s3.amazonaws.com/clickhouse-test-reports/75670/0a472ed2df545463c1443425cd55f2d9ab4a6d6f/stress_test__debug_.html) — No errors found
>[Stress test (tsan)](https://s3.amazonaws.com/clickhouse-test-reports/75670/0a472ed2df545463c1443425cd55f2d9ab4a6d6f/stress_test__tsan_.html)
Stress test (tsan) — No errors found
>[Stress test (ubsan)](https://s3.amazonaws.com/clickhouse-test-reports/75670/0a472ed2df545463c1443425cd55f2d9ab4a6d6f/stress_test__ubsan_.html)
Stress test (ubsan) — No errors found

Looks hood

azat committed:
ci/stress: disable cannot_allocate_thread_fault_injection_probability for final check

CI: https://s3.amazonaws.com/clickhouse-test-reports/75670/0a472ed2df545463c1443425cd55f2d9ab4a6d6f/stress_test__tsan_.html
